### DISCUSSION TITLE
Kézi nyugta lehetősége és kézi nyugtával kapcsolatos adatszolgáltatás

### DISCUSSION THREAD

**Discussion by wlln22** [nav-gov-hu/eRECEIPT#33] [Kérdések és válaszok]
A rendelettervezet 7-es és 8-as mellékletben nem érintett vállalkozások számára a kézi nyugta továbbra is választható opció? Kézi nyugtáról lesz-e valamilyen kötelező adatszolgáltatás? Van terv arra, hogy a jövőben tovább bővüljön a pénztárgépre kötelezettek köre a rendeletben említetteken kívül?

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
